frontend/src/model/beta/images/UploadNewImageDialog.tsx

```diff
@@ -0,0 +1,127 @@
+    setDisplayedToken(token)
+    setDisplayToken(true)
+  }
+
+  if (isUiConfigError) {
+    return <MessageAlert message={isUiConfigError.info.message} severity='error' />
+  }
+
+  if (isCurrentUserError) {
+    return <MessageAlert message={isCurrentUserError.info.message} severity='error' />
+  }
+
+  return (
+    <>
+      {(isUiConfigLoading || isCurrentUserLoading) && <Loading />}
+      {uiConfig && currentUser && (
+        <Dialog open={open} onClose={handleClose}>
+          <DialogTitle color='primary'>Uploading an Image for this Model</DialogTitle>
+          <DialogContent>
+            <Stack spacing={2}>
+              <Typography fontWeight='bold'>User authentication token</Typography>
+              <Typography>Use the token below to authenticate when you try and run the docker login command</Typography>
+              <Stack direction='row'>
+                <Button sx={{ mr: 2 }} variant='outlined' onClick={showToken} data-test='showTokenButton'>
+                  Regenerate Token
+                </Button>
+                <Box
+                  sx={{
+                    backgroundColor: theme.palette.container.main,
+                    pr: 2,
+                    pl: 2,
+                    display: 'flex',
+                    mr: 1,
+                  }}
+                >
+                  <Box component={Stack} direction='column' justifyContent='center'>
+                    <Typography variant='body1' data-test='dockerPassword'>
+                      {displayToken ? displayedToken : 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxxx'}
+                    </Typography>
+                  </Box>
+                </Box>
+                <Tooltip title='Regenerate & Copy to clipboard'>
+                  <IconButton
+                    onClick={async () => {
+                      const token = await regenerateToken()
+                      setDisplayedToken(token)
+                      navigator.clipboard.writeText(token)
+                    }}
+                    aria-label='regenerate and copy to clipboard'
+                  >
+                    <ContentCopy />
+                  </IconButton>
+                </Tooltip>
+              </Stack>
+              <Stack spacing={1}>
+                <Typography fontWeight='bold'>Logging in</Typography>
+                <Stack spacing={2}>
+                  <CodeLine line={`docker login ${uiConfig.registry.host} -u ${currentUser.id}`} />
+                </Stack>
+              </Stack>
+              <Stack spacing={1}>
+                <Typography fontWeight='bold'>Pushing an image to the registry</Typography>
+                <Stack spacing={2}>
+                  <CodeLine line={`docker tag <image> ${window.location.host}/<namespace>/<model>:<version>`} />
+                  <CodeLine line={`docker push ${window.location.host}/<namespace>/<model>:<version>`} />
+                  <CodeLine line={`${window.location.host}/api/v2/model/myModelId/images`} />
```

Comment:
This isn't a bash command?

Reply:
Ah no this is the URL for listing all the images, I forgot to remove the dollar sign. I'll see if I can also make it clearer.

Reply:
Ah okay. Yes, making it clear that it is separate from the commands used to push to the registry sounds like a good idea as well. Could you also add copy buttons to the bash commands?

Reply:
You can actually click the lines to copy them, and a tooltip appears like so:

![image](https://github.com/gchq/Bailo/assets/89992537/1b3d3a32-4465-42bb-9600-705c4bb7fcb8)

I've also added some icons to the left of each to make it clearer

Reply:
I don't think we need to list the API endpoint within our frontend?